internal/server/openapi.go

```diff
@@ -430,6 +431,20 @@ func buildResponse(schemas openapi3.Schemas, rst reflect.Type) openapi3.Response
 func buildRequest(r reflect.Type, op *openapi3.Operation) {
 	op.Parameters = openapi3.NewParameters()
 
+	op.AddParameter(&openapi3.Parameter{
+		Name:     "Infra-Version",
+		In:       "header",
+		Required: true,
+		Schema: &openapi3.SchemaRef{
+			Value: &openapi3.Schema{
+				Example:     internal.FullVersion(),
```

Comment:
This is going to re-introduce the problem that was fixed in #2428. Every time we do a release, and every time this ends up on the `release-please` branch, this version will change, which will cause the `TestWriteOpenAPIDocToFile` test to fail.

We need some way to pass in the version so that we don't rely on the static `internal.FullVersion`.

If we can't pass in the version, we'll have to patch it in `TestWriteOpenAPIDocToFile` to set it to a hardcoded value like `0.0.0`.